javascript-constructors/student.js

```diff
@@ -0,0 +1,14 @@
+/* exported Student */
+function Student(firstName, lastName, subject) {
+  this.firstName = firstName;
+  this.lastName = lastName;
+  this.subject = subject;
+};
+
+Student.prototype.getFullName = function () {
+  return this.firstName + ' ' + this.lastName;
+};
+
+Student.prototype.getIntroduction = function () {
+  return 'Hello, my name is Grady Booch and I am studying ' + this.subject + '.';
```

Comment:
Please make sure this gives the current student's name, even if they're not named "Grady Booch".